### PR TITLE
unreadCount event

### PR DESCRIPTION
**Pull request by fsalomon**
Usage:
```
Smooch.on('unreadCount', function(unreadCount) {
    console.log('the number of unread messages was updated', unreadCount);
});
```

### CODE REVIEW

**Comment by lemieux** [comment]
It will also need an addition to this section in the readme https://github.com/smooch/smooch-js/blob/integration/README.md#events

File: src/js/components/message-indicator.jsx [hunk 1]
```diff
@@ -70,6 +71,7 @@ export class MessageIndicatorComponent extends Component {
     }
 
     componentWillReceiveProps({unreadCount}) {
+        observable.trigger('unreadCount', unreadCount);
```
Comment (lemieux) [inline src/js/components/message-indicator.jsx]: This should be triggered here https://github.com/smooch/smooch-js/blob/integration/src/js/smooch.jsx#L102 instead right after `handleNotificationSound();`.

**Comment by fsalomon** [comment]
Thanks! Changes done.

**Comment by lemieux** [comment]
Just to let you know it went out yesterday with the 3.13.0 release. Thanks again for your contribution!

**Pull request merged**